types/nightwind/test/nightwind__helper-tests.global.ts

```diff
@@ -0,0 +1,15 @@
+import * as nightwind from "nightwind/helper";
```

Comment:
Global tests shouldnt have imports

Reply:
true, those tests should not have been `.global`